download-pyega3/main.nf

```diff
@@ -70,8 +70,9 @@ process downloadPyega3 {
     export PYEGA3_EGA_USER=${params.pyega3_ega_user}
     export PYEGA3_EGA_PASS=${params.pyega3_ega_pass}
     mkdir -p ${ega_id}
-    python3.6 /tools/main.py \\
+    python3 /tools/main.py \\
     	-f ${ega_id} \\
+      -c ${params.cpus} \\
```

Comment:
According to [the note](https://github.com/EGA-archive/ega-download-client#slow-download-speeds), it is recommended to use `30` as a starting point and do some adjustments. We probably do not need to relate it with cpus.

Reply:
We can expose`--connections` as a separate param.